lms/djangoapps/teams/models.py

```diff
@@ -15,6 +15,7 @@ class CourseTeam(models.Model):
     """This model represents team related info."""
 
     team_id = models.CharField(max_length=255, unique=True)
```

Comment:
After working on #9009, I do think we should rename discussion_id. How about discussion_topic_id or commentable_id?.